frame/request/query.go

```diff
@@ -16,6 +16,7 @@ type Query struct {
 func (q *Query) WriteTo(b *frame.Buffer) {
```

Comment:
Cheers for clarifications in the commit message! 
Can you please add one line separator between header and body.
The reason is that Goland is stupid 

<img width="1033" alt="image" src="https://user-images.githubusercontent.com/1617930/150817482-aeb93f77-ff8f-4aae-b950-70fdae64237a.png">